src/python/pants/bin/loader_integration_test.py

```diff
@@ -82,3 +82,16 @@ def run(limit: str) -> PantsResult:
     assert "RecursionError" in small_run.stderr
     # Non integer value fails.
     run("this isn't an int").assert_failure()
+
+
+def test_non_utf8_env_vars() -> None:
+    res = run_pants(
+        command=["--version"],
+        extra_env={
+            "FOO": b"B\xa5R",
+            b"F\xa5O": "BAR",
+        },
+        use_pantsd=True,
+    )
+    print(res.stdout)
```

Comment:
Is this `print` left over? And/or are we intending to validate that the output references the problematic variables?

Reply:
Ooops, yes. Debug print.